src/gui/GuiRunner.hh

```diff
@@ -49,6 +49,7 @@ class IGNITION_GAZEBO_GUI_VISIBLE GuiRunner : public QObject
   public: ~GuiRunner() override;
 
   /// \brief Callback when a plugin has been added.
+  /// This function has no effect and is left here for ABI compatibility.
```

Comment:
Since we don't have to worry about API/ABI breaks in fortress, should this method be removed?

Reply:
Nevermind, I see that this was asked/answered already in https://github.com/ignitionrobotics/ign-gazebo/pull/951#discussion_r681152457